fluster/test.py

```diff
@@ -69,7 +69,11 @@ def _test(self) -> None:
         input_filepath = os.path.join(
             self.resources_dir,
             self.test_suite.name,
-            self.test_vector.name,
+            (
+                self.test_vector.name
+                if self.test_suite.is_single_archive is False
```

Comment:
```suggestion
                if not self.test_suite.is_single_archive
```

Reply:
Black doesn't like this. Maybe when we merge the ruff PR this will be allowed